packages/matchbox/src/hooks/useInputDescribedBy.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+/**
+ * Generates an aria-describedby attribute and ids for HelpText and Error if id is present
+ *
+ * @example
+ *  const { describedBy, errorId, helpTextId } = useInputDescribedBy({
+ *    id: 'input-id',
+ *    helpText: 'This is helptext',
+ *    error: 'This is an error'
```

Comment:
It doesn't look like the actual help text is used in the hook? Just that the help text exists? Could these be booleans like `hasHelpText` and `hasErrorText`?

Reply:
Yeah good call 👍